projects/croniter/fuzz_iter.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/python3
-# Copyright 2022 Google LLC
+# Copyright 2023 Google LLC
```

Comment:
No need to bump the year -- we keep the copyright from when it was introduced.

Reply:
Copyright year reverted back to `2022`